src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -85,4 +90,139 @@ std::optional<SystemMemoryUsage> GetSystemMemoryUsage() noexcept {
   return parsing_result;
 }
 
+int64_t ParseProcessStatus(const std::string& status_content) {
```

Comment:
Nit: const std::string& should be string_view.

Reply:
Thanks for bringing this to my attention! Changed accordingly.